src/opnsense/service/modules/addons/template_helpers.py

```diff
@@ -151,15 +151,27 @@ def physical_interfaces(self, names):
             result.append(self.getNodeByTag('interfaces.'+name+'.if'))
         return list(filter(None, result))
 
-
-    def get_bracketed_ip(self, str):
-        """ returns ipv6 addresses with brackets else returns original
-        :param str: any string that may be an IPv6 address in a place where an IPv6 address would need brackets
+    def get_host_port(self, host: str, port: str | int):
```

Comment:
Since the usage is in a plugins PR I've made a comment there:

https://github.com/opnsense/plugins/pull/4823/files/4022932e935be6606694290868218c4961db4468..cb1ee952723967a0ddd5582bd23f2d1a72fb0f71#r2215689666